### PR TITLE
Add product vision/scope.

### PR DESCRIPTION
**Pull request by fniessink**

### CODE REVIEW

File: docs/src/intro.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ Implemented features include:
 - Export of reports to PDF, both via the UI as well as via the API.
 - Notifications of events, such as metrics turning red, to Microsoft Teams.
 - Side-by-side comparison of measurements at different points in time.
+- Integration with issue tracker (Jira only at the moment) to manage actions and technical debt.
```
Comment (wkoot) [inline docs/src/intro.md:99]: Azure DevOps issues/work items are also integrated

File: docs/src/intro.md [hunk 2]
```diff
@@ -125,3 +126,21 @@ docker-compose up
 ```
 
 *Quality-time* is served at [http://localhost](http://localhost). Use username `jadoe` and password `secret` to log in.
+
+# Why *Quality-time*?
+
+ICTU developed *Quality-time* to help projects and teams within ICTU gain actionable insight into the quality of the software they are developing and maintaining. The current vision on the scope of *Quality-time* is:
+
+- *Quality-time* collects data from sources that projects and teams are using anyway to develop and maintain software. *Quality-time* does not analyse the software itself.
```
Comment (wkoot) [inline docs/src/intro.md]: the word "anyway" seems redundant, how about:
`that projects and teams use in developing and maintaining software`

File: docs/src/intro.md [hunk 3]
```diff
@@ -125,3 +126,21 @@ docker-compose up
...
+- The sources that *Quality-time* uses to collect its measurement data are automated. Manual sources for measurements are possible, but automated sources are preferred.
```
Comment (wkoot) [inline docs/src/intro.md:135]: Isn't it more the source data that's automatically generated?

File: docs/src/intro.md [hunk 4]
```diff
@@ -125,3 +126,21 @@ docker-compose up
...
+- *Quality-time* evaluates measurement data against target values. It shows users which metrics do not meet their target value, so users can take corrective action. *Informational* metrics, meaning metrics without target, are possible, but most metrics are assumed to have targets.
+- *Quality-time* provides a single overview of the software measurements, but refers users to the underlying tools for detailed information.
+- Technical debt management is an integral part of software quality management. Any metric that does not meet its target value for a longer period of time can be considered to represent a form of debt that needs to be managed.
+- Tools come and go. Hence, an important non-functional requirement for *Quality-time* is the ability to quickly add interfacing to new sources. However, it is not a requirement to make this possible without developing a small amount of interface code.
```
Comment (wkoot) [inline docs/src/intro.md]: "not a requirement .. without" is a double negative, how about:
`For this purpose, developing and maintaining a small amount of interface code is acceptable.`

File: docs/src/intro.md [hunk 5]
```diff
@@ -125,3 +126,21 @@ docker-compose up
...
+- *Quality-time* provides actionable information on current quality issues and risks. Historical information is retained, but is a second-class citizen.
+- *Quality-time* is not an issue tracker or task manager. However, it does integrate with issue trackers so it's easy to create issues for metrics that need action and to manage technical debt.
```
Comment (wkoot) [inline docs/src/intro.md]: Easier reading:
`It does however integrate with issue trackers, so`

File: docs/src/intro.md [hunk 6]
```diff
@@ -125,3 +126,21 @@ docker-compose up
...
+- Software quality is usually subdivided into quality characteristics such as maintainability, testability, and accessibility, as for example in ISO 25010. Many metrics in *Quality-time* measure parts of these quality characteristics. For example, the number of 'code smells' and the 'complexity' of source code are both related to maintainability. However, *Quality-time* does not attempt to aggregate these metrics into a measurement of these quality characteristics. Such aggregations typically lack a sound mathematical and scientific basis and are often not actionable.
```
Comment (wkoot) [inline docs/src/intro.md]: Hmm, if it is subdivided then what was the original division?

File: docs/src/intro.md (hunk 1 above)
Comment (fniessink) [reply docs/src/intro.md:99]: Unfortunately not, only Jira can be configured as issue tracker.

File: docs/src/intro.md (hunk 3 above)
Comment (fniessink) [reply docs/src/intro.md:135]: That's true, that's why it's an automated source: it generates data in an automated manner ;-)

File: docs/src/intro.md (hunk 4 above)
Comment (fniessink) [reply docs/src/intro.md]: I'll just remove that sentence. Explaining all that properly takes too many words.

File: docs/src/intro.md (hunk 5 above)
Comment (fniessink) [reply docs/src/intro.md]: Solved differently.

File: docs/src/intro.md (hunk 6 above)
Comment (fniessink) [reply docs/src/intro.md]: I was looking for the English word for "onderverdeeld". Maybe just "divided" is better. Or "broken down".

File: docs/src/intro.md (hunk 2 above)
Comment (fniessink) [reply docs/src/intro.md]: It's not redundant IMO. Compare to the Dutch: "Quality-time verzamelt gegevens uit bronnen die projecten en team sowieso gebruiken om software te ontwikkelen en onderhouden". Anyway/sowieso stress that the (data in the) tools are reused for the purpose of quality management.

**Pull request merged**